content/en/continuous_integration/troubleshooting.md

```diff
@@ -20,6 +20,10 @@ kind: documentation
 3. For languages other than Swift, make sure the Datadog Agent is running on the host where tests are run (accessible at `localhost:8126`), or if accessible on another hostname or port, make sure you run your tests with the appropriate Agent hostname set in the `DD_AGENT_HOST` and the appropriate port in `DD_TRACE_AGENT_PORT` environment variables. You can activate [debug mode][5] in the tracer to check if it's able to connect to the Agent.
 4. If you still don't see any results, [contact Support][1] for troubleshooting help.
 
+### You have received a "We couldn't find this pipeline" message
```

Comment:
Using some of the feedback from our copy guidelines, can we avoid the usage of "We"?

Instead something like, "Pipeline not found"

We'd need to update the message in the UI as well to match cc @soyguijarro

Reply:
Fixed